src/prototype.rs

```diff
@@ -169,10 +169,6 @@ impl<'a> FirstPass<'a> {
             return self.parse_html_block_type_6_or_7(ix, remaining_space);
         }
 
-        // Remaining blocks aren't impacted by leading spaces, so we can consume
```

Comment:
If you want to fix the initial whitespace sourcemap issue, the way to do it is: leave this here, remove the `ix` advance above (before the html blocks), _save_ the line_start (by cloning) before the `let indent = line_start.scan_space_upto(4)`, and pass in the non-advanced `ix` and the remaining space from the saved line_start. (Alternatively, now that I think about it, you can save just the remaining_space).

As I say, this is a small issue, but I'm trying to think about the "right way" to do these kinds of things.